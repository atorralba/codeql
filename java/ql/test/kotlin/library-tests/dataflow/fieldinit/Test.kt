class Foo {
    val prop: String = "src"

    fun bar() {
       sink(prop)
    }
}

fun sink(s: String) {}