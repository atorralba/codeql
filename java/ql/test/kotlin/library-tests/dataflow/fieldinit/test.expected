| Test.kt:2:24:2:28 | src | Test.kt:5:13:5:16 | getProp(...) |
